python/katana/dynamic_bitset.pyx

```diff
@@ -0,0 +1,73 @@
+from .cpp.libgalois.DynamicBitset cimport DynamicBitset as CDynamicBitset
+
+
+cdef class DynamicBitset:
+    cdef CDynamicBitset underlying
+
+    def __init__(self, num_bits):
+        self.resize(num_bits)
+
+    def resize(self, n):
+        """Resizes the bitset."""
+        self.underlying.resize(n)
```

Comment:
Argument names are part of the API for users since Python has keyword arguments and the names are used in docs. So use the same argument name for `__init__` and `resize`.